src/explore-education-statistics-common/src/styles/utils/_text.scss

```diff
@@ -1,3 +1,7 @@
 .dfe-white-space--pre-wrap {
   white-space: pre-wrap;
 }
+
+.dfe-word-wrap--break-word {
```

Comment:
Shouldn't this be `dfe-word-break--break-word`?

Reply:
doh, yes. Sorted now.